### PR TITLE
completed sql migration homework

### PR DESCRIPTION
**Pull request by randy525**

### CODE REVIEW

File: src/main/resources/db/migration/V20191004032800__create_table_pay.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE pay
+(
+    cardNr VARCHAR(16) PRIMARY KEY,
+    employee_id NUMBER(6) NOT NULL,
+    salary NUMBER(8, 2) NOT NULL,
+    commission_pct NUMBER(2, 2)
+);
```
Comment (icovdii) [inline src/main/resources/db/migration/V20191004032800__create_table_pay.sql]: It seems like you missed here one condition from task requirements: salary and commission_pct should be transferred from employees table. Don't see any actions related to it, or may be I just missed them

File: src/main/resources/db/migration/V20191004033100__create_table_projects.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    project_investment NUMBER(10, -3) NOT NULL ,
+    project_revenue NUMBER(10, -3),
+    CONSTRAINT prjct_description_min CHECK ( LENGTH(project_description) > 10 ),
+    CONSTRAINT prjct_investment_min CHECK ( project_investment > 0 )
+);
+
+CREATE SEQUENCE projects_seq START WITH 100 INCREMENT BY 2 NOCACHE;
+
+ALTER TABLE projects
+    MODIFY project_id DEFAULT projects_seq.nextval;
```
Comment (icovdii) [inline src/main/resources/db/migration/V20191004033100__create_table_projects.sql:14]: Where is the many-to-many relation with employees table?

Comment (randy525) [reply src/main/resources/db/migration/V20191004033100__create_table_projects.sql:14]: I implemented this relation using the hours_worked table, which has project_id and employee_id

Comment (icovdii) [reply src/main/resources/db/migration/V20191004033100__create_table_projects.sql:14]: It's ok for this home work, I think you can leave it as it is, but just for future: take into account that feature or fix you implement as a developer must correspond completely to the details in task/user story you are working on. If by some reasons a requirement can not be implemented it should be discussed with teamLead/manager/client and this details will be documented by comments in user story.

**Review by icovdii — APPROVED** [review]

File: src/main/resources/db/migration/V20191004032800__create_table_pay.sql (hunk 1 above)
Comment (randy525) [reply src/main/resources/db/migration/V20191004032800__create_table_pay.sql]: Discussed

**Pull request merged**